### PR TITLE
Type definitions for local-storage

### PR DESCRIPTION
**Pull request by alex7229**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/local-storage/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+    /** https://github.com/bevacqua/local-storage#lsonkey-fn */
+    on(key: string, callback: Listener): void;
+
+    /** https://github.com/bevacqua/local-storage#lsoffkey-fn */
+    off(key: string, callback: Listener): void;
+}
+
+declare const accessor: Accessor;
+export default accessor;
```
Comment (plantain-00) [inline types/local-storage/index.d.ts]: It's not `export default` https://github.com/bevacqua/local-storage/blob/master/local-storage.js#L42

**Review by plantain-00 — CHANGES_REQUESTED** [review]

File: types/local-storage/index.d.ts (hunk 1 above)
Comment (tup1tsa) [reply types/local-storage/index.d.ts]: Done.

**Review by plantain-00 — APPROVED** [review]

File: types/local-storage/local-storage-tests.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import * as ls from 'local-storage';
```
Comment (armanio123) [inline types/local-storage/local-storage-tests.ts]: Namespaces are not callable, instead use `import name = require='module'`

**Review by armanio123 — CHANGES_REQUESTED** [review]

File: types/local-storage/local-storage-tests.ts (hunk 2 above)
Comment (tup1tsa) [reply types/local-storage/local-storage-tests.ts]: My bad

**Review by armanio123 — APPROVED** [review]

**Pull request merged**